distribution/changelog.txt

```diff
@@ -24,6 +24,7 @@
 - Fix: [#24403] Park fences draw underneath and through opaque water.
 - Fix: [#24406] The network status window uses an undefined string for its title.
 - Fix: [#24444] In the object load error window, the guide text overlaps when the title bar is enlarged.
+- Fix: [#24447] Shortcuts are not refreshed when changing language.
```

Comment:
```suggestion
- Fix: [#24447] Shortcut window list is not refreshed when changing language.
```

Reply:
Actually the shortcut description is refreshed already, just the actual shortcuts that are not.